nvm.sh

```diff
@@ -1044,6 +1044,20 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
+    MAKE_JOBS="$(nproc)"
+  elif [ "_$NVM_OS" = "_freebsd" ] || [ "_$NVM_OS" = "_darwin" ]; then
+    MAKE_JOBS="$(sysctl -n hw.ncpu)"
+  elif [ "_$NVM_OS" = "_sunos" ]; then
+    MAKE_JOBS="$(psrinfo | wc -l)"
+  fi
+  if [ -z "$MAKE_JOBS" ]; then
+    echo "Can not determinate how many thread(s) we can use, set to only 1 now."
+    echo "Please report an issue on GitHub to help us make it better and run it faster on your computer!"
```

Comment:
I can play with the wording later ("determine", not "determinate") and I'll want it sent to stderr, not stdout - but this is great, thanks :-)